edx_dl/parsing.py

```diff
@@ -388,7 +388,7 @@ def _make_subsections(section_soup):
             # FIXME correct extraction of subsection.name (unicode)
             subsections = [SubSection(position=i,
                                       url=s.a['href'],
-                                      name=s.a.div.span.string.strip())
+                                      name=s.a.div.div.string.strip())  # modified by kyilmaz80
                            for i, s in enumerate(subsections_soup, 1)]
```

Comment:
Again, no need to annotate the change with your name. Anyway, does extraction of courses from Stanford and from "regular" edX still work after your changes?

Reply:
I tested with stanford case "-x stanford --list-courses". It is working fine. Also I was able to download a course.

Reply:
OK, good.